assets/js/script.js

```diff
@@ -0,0 +1,28 @@
+class Book {
+  constructor() {
+    this.bookArray = Object.entries(window.localStorage);
+    this.html = [];
+  }
+
```

Comment:
- [ ] From the [project description](https://github.com/microverseinc/curriculum-javascript/blob/main/books/m1_plain_js_objects.md) you are not supposed to use classes for this milestone. Rather, you are to use plain objects and javascript functions to add, remove and display books. Classes will be implemented in the next milestone. Kindly work on this

Reply:
This issue is well resolved ✔️